addons/website_hr_recruitment/models/hr_recruitment.py

```diff
@@ -45,11 +45,11 @@ def _get_default_website_description(self):
         return (default_description.render() if default_description else "")
 
     website_description = fields.Html('Website description', translate=html_translate, sanitize_attributes=False, default=_get_default_website_description, prefetch=False)
-
     def _compute_website_url(self):
         super(Job, self)._compute_website_url()
         for job in self:
-            job.website_url = "/jobs/detail/%s" % job.id
+            # can't publish a job of company without website
+            job.website_url = job.company_id.website and "/jobs/detail/%s" % job.id
```

Comment:
? website is a plain text field, it has nothing to do and it not and should not be involved in anything regarding the website module